packages/components/radio-group/src/radio-group.js

```diff
@@ -0,0 +1,134 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './radio-group.css';
+import { getDirectionByKey, customEventNames } from './utils';
+
+import '@tradeshift/elements.radio';
+
+customElementDefineHelper(
+	'ts-radio-group',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				dir: { type: String, reflect: true },
+				focused: { type: Boolean, reflect: true },
+				index: { type: Number, reflect: true },
+				value: { type: String, reflect: true },
+				title: { type: String, reflect: true }
+			};
+		}
+
+		get direction() {
+			return this.dir ? this.dir : this.bodyDir;
+		}
+
+		toggle(radioNode) {
+			if (!radioNode || radioNode.disabled) {
+				return;
+			}
+
+			for (const radio of this.radios) {
+				radio.checked = radio === radioNode;
+				if (radio.checked) {
+					this.index = this.radios.indexOf(radio);
+					this.value = radio.value;
+				}
+			}
+		}
+
+		firstUpdated() {
+			this.shadowRoot.host.addEventListener(customEventNames.RADIO_CLICK, this.handleRadioClick);
+			this.shadowRoot.host.addEventListener(customEventNames.RADIO_FOCUS, this.handleFocus);
+			this.shadowRoot.host.addEventListener(customEventNames.RADIO_BLUR, this.handleBlur);
+			this.handleSlotChange();
+		}
+
+		disconnectedCallback() {
+			this.shadowRoot.host.removeEventListener(customEventNames.RADIO_CLICK, this.handleRadioClick);
+			this.shadowRoot.host.removeEventListener(customEventNames.RADIO_FOCUS, this.handleFocus);
+			this.shadowRoot.host.removeEventListener(customEventNames.RADIO_BLUR, this.handleBlur);
+		}
+
+		handleRadioClick(e) {
+			this.toggle(e.target);
+		}
+
+		handleFocus() {
+			this.focused = true;
+		}
+
+		handleBlur() {
+			this.focused = false;
+		}
+
+		handleSlotChange() {
+			this.radios = Array.from(this.shadowRoot.host.querySelectorAll('ts-radio'));
+
+			for (let i = 0; i < this.radios.length; i++) {
+				const radio = this.radios[i];
+				if (radio.checked && !radio.disabled) {
+					this.index = i;
+				}
+			}
+			if (this.index) {
+				this.toggle(this.radios[this.index]);
+			}
+		}
+
+		handleKeyDown(e) {
+			if (this.radios.length === 0) {
+				return;
+			}
+
+			const enabled = this.radios.filter(checkbox => !checkbox.disabled);
+			if (enabled.length === 0) {
+				return;
+			}
+
+			const direction = getDirectionByKey(e.code);
+			if (!direction) {
+				return;
+			}
+
+			const getNextCheckbox = direction => {
+				this.index = this.index + direction;
+				if (this.index <= -1) this.index = this.radios.length - 1;
+				if (this.index >= this.radios.length) this.index = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)